### ISSUE TITLE
 cannot find symbol RectF bounds = new RectF();

### ISSUE THREAD

**Issue by ebrugulec** [react-native-community/react-native-svg#778]
I am trying to add react-native-svg with npm. But i am getting this error.

```
Task :react-native-svg:compileDebugJavaWithJavac
C:\Users\ebugl\sample\node_modules\react-native-svg\android\src\main\java\com\horcrux\svg\TSpanShadowNode.java:853: error: cannot find symbol
            RectF bounds = new RectF();
            ^
  symbol:   class RectF
  location: class TSpanShadowNode
C:\Users\ebugl\sample\node_modules\react-native-svg\android\src\main\java\com\horcrux\svg\TSpanShadowNode.java:853: error: cannot find symbol
            RectF bounds = new RectF();
                               ^
  symbol:   class RectF
  location: class TSpanShadowNode
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\Users\ebugl\sample\node_modules\react-native-svg\android\src\main\java\com\horcrux\svg\RenderableView.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
```

![photofacefun_com_1537033392](https://user-images.githubusercontent.com/10262499/45589063-0051ee80-b928-11e8-8131-39e77c059f87.jpg)

Can you help me? How can i fix?

**Comment by msand** [comment]
@ebrugulec 😲 My mistake. Thanks for the issue report! I've fixed the issue and published v7.0.1 please install that version and try again ❤️

**Comment by ebrugulec** [comment]
> @ebrugulec 😲 My mistake. Thanks for the issue report! I've fixed the issue and published v7.0.1 please install that version and try again ❤️

Thanks for the quick response on handling this situation. 😇 It is working now 🌼
